src/components/PartnerHero/PartnerHero.tsx

```diff
@@ -85,7 +85,8 @@ export const PartnerHero: FC<PartnerHeroProps> = ({
                 }
                 openSeaLink={headline.openSeaLink}
                 interactWithNFTLink={headline.interactWithNFTLink}
-                dataSuffix={headline.dataSuffix || siteDataSuffix}
```

Comment:
I took this away because by default the checkbox is unchecked. I interpreted that as dataSuffix should always be set to siteDataSuffix unless the user checks the checkbox then the value switches to the one on the drop config. Let me know if this is incorrect then I can put this line back in.

Reply:
ah I see. Yea there's a few ways to make this all work. But this makes some sense with box starting unchecked. Tbh it doesn't even have to be in the Drop config but could just be a checkbox hook like `if drop.address == X, use this other specific hex`, but OK with this if it's all working. Have we tested when box NOT checked? @ximxim

Reply:
I'm just not sure we need to be passing two things around

Reply:
Yes it works, I have tested it locally. Just wanted to avoid hardcoding but if that's the approach you want me to take, I can change things around. Do let me know.